h/static/scripts/util/string.js

```diff
@@ -19,7 +24,39 @@ function unhyphenate(name) {
   }
 }
 
+
+/**
+ * Utility function to normalize a string based on our standard pattern of NFKD
+ *
+ * @param  {String}
+ * @returns {String}
+ */
+function normalize(str){
+
+  // This require is coming from a vendor bundle
+  // that is loaded globally on the running webpage
+  // not a require in the node context
+  if(!require || !require('unorm') || !String.prototype.normalize){
```

Comment:
Couple of issues:

1. If `require` is not defined, this would result in a `ReferenceError` exception because of strict mode. Given that we are using CommonJS, I think we can just assume the existence of require as a function here, so we can just remove that check.

2. If `require()` fails here, I think it would throw an exception rather than returning false (but please check, I may be mistaken), so you'd need to use `try..catch`.